### PR TITLE
Fix for deep nested v-layout's being affected by grid list styling

### PR DESCRIPTION
**Pull request by Phlow2001**
Example of problem that this solves:  https://codepen.io/jayblanchard/pen/LzXEQw?editors=1000

### CODE REVIEW

File: src/stylus/components/_grid.styl [hunk 1]
```diff
@@ -29,17 +29,17 @@
       &-{$size}
         padding: $gutter
 
-        .layout
+        > .layout
           .flex
```
Comment (KaelWD) [inline src/stylus/components/_grid.styl]: I'd probably add one here as well.

**Comment by johnleider** [comment]
This is how it was in the past, I removed it but did not leave a comment, I'll need to dig it back up.

**Comment by johnleider** [comment]
This breaks the ability to do nested grids. https://codepen.io/johnjleider/pen/veMKEL?&editors=101 as an example.

**Comment by Phlow2001** [comment]
It's a grid-list-xl, there's no gutters.

**Comment by johnleider** [comment]
I'm referring to the example codepen that I posted, which is a common functionality that we need to support.

**Comment by Phlow2001** [comment]
Sorry, to be clear, my response "It's a grid-list-xl, there's no gutters." is to your statement "I am not entirely sure what the issue is with the codepen you provided.".

We've talked about it before, I just wanted to record it in the issue in case you forgot.

**Comment by dohomi** [comment]
@Phlow2001 @johnleider I just bumped into this issue as well and fixed it with this PR. Alternative users can set `.ma-0` to the nesting layout as follow:
https://codepen.io/anon/pen/rYyBZN?editors=1000

**Comment by johnleider** [comment]
@dohomi's work around will be the preferred method for this case, as implementing this PR breaks other use-cases that we already support. Thank you for your contribution.

**Pull request closed without merging**